en/08_Changelogs/6.0.0.md

```diff
@@ -4,61 +4,63 @@ title: 6.0.0 (unreleased)
+  - [Changes to `sake`, `BuildTask`, and CLI interaction in general](#cli-changes)
   - [Read-only replica database support](#db-read-only-replicas)
+  - [Improvements to template functionality](#improvements-to-template-functionality)
   - [Performance improvements for site tree rendering](#site-tree-performance)
-  - [Run `CanonicalURLMiddleware` in all environments by default](#url-middleware)
   - [Changes to default cache adapters](#caching)
-  - [Changes to scaffolded form fields](#scaffolded-fields)
-  - [`SiteTree` uses form field scaffolding](#sitetree-scaffolding)
   - [Filter within a range with `searchable_fields`](#searchable-withinrange)
-  - [Changes to the templating/view layer](#view-layer)
-  - [Changes to `LeftAndMain` and its subclasses](#leftandmain-refactor)
-  - [Changes to password validation](#password-validation)
   - [Status flags in the CMS](#status-flags-in-the-cms)
   - [New `Versioned` methods](#new-versioned-methods)
-  - [New default front-end theme](#theme)
   - [Other new features](#other-new-features)
-- [Dependency changes](#dependency-changes)
-  - [`intervention/image` has been upgraded from v2 to v3](#intervention-image)
-  - [symfony dependencies have been upgraded from v6 to v7](#symfony)
-  - [JavaScript dependencies](#javascript-dependencies)
+- [Breaking changes](#breaking-changes)
+  - [Changes to scaffolded form fields](#scaffolded-fields)
+  - [`SiteTree` uses form field scaffolding](#sitetree-scaffolding)
+  - [MySQL now defaults to utf8mb4](#mysql-utf8mb4)
+  - [Run `CanonicalURLMiddleware` in all environments by default](#url-middleware)
+  - [Session and authentication cookie changes](#session-cookie-changes)
+  - [`DBDecimal` default value](#dbdecimal-default-value)
+  - [`RedirectorPage` validation](#redirectorpage-validation)
+  - [Update JS MIME type, remove `type` in `<script>` tags](#js-mime-type-update)
+  - [`getSchemaDataDefaults()` now includes attributes](#formfield-schema-data)
+  - [Remember me token rotation](#remember-me-token-rotation)
 - [Bug fixes](#bug-fixes)
 - [API changes](#api-changes)
   - [Many renamed classes](#renamed-classes)
+  - [Changes to some extension hook names](#hooks-renamed)
   - [GraphQL removed from the CMS](#graphql-removed)
+  - [Changes to the templating/view layer](#view-layer)
+  - [Changes to `LeftAndMain` and its subclasses](#leftandmain-refactor)
   - [`FormField` classes now use `FieldValidator` for validation](#formfield-validation)
   - [`CMSMain` search filter changes](#cmsmain-search-filter)
   - [Most extension hook methods are now protected](#hooks-protected)
-  - [Changes to some extension hook names](#hooks-renamed)
   - [`FormField::Value()` split into two methods](#formfield-value)
   - [`Controller::has_curr()` removed](#controller-has-curr)
   - [Strict typing for `Factory` implementations](#factory-strict-typing)
-  - [Elemental `TopPage` class names changed](#elemental-top-page)
```

Comment:
1. Removed "Elemental `TopPage` class names changed" section and put that content straight into "Many renamed classes".
1. Moved "changes to some extension hook names" right under "Many renamed classes" so the renaming stuff is grouped
1. Moved "Changes to the templating/view layer" and "Changes to `LeftAndMain` and its subclasses" into the API changes section because they're _more_ about changes to the API than they are about functional enhancements